qa-open-login.php

```diff
@@ -99,18 +99,21 @@ function check_login()
 			if (!isset($topath)) {
 				$topath = ''; // redirect to front page
 			}
-
+			
+			// try to login
+			$hybridauth = new Hybrid_Auth($config);

```

Comment:
Any reason to move this out of the try block?

Reply:
My first login hit the `catch` block and it freaked out b/c the `$adapter` wasn't defined

Reply:
But won't you still get an exception?

Reply:
No, the `catch` just redirects you back to the homepage

Reply:
https://github.com/RedVentures/q2a-open-login/blob/changeHandleValue/qa-open-login.php#L129-L142